.circleci/config.yml

```diff
@@ -858,6 +860,7 @@ jobs:
 
   Python Windows i686 tests:
     docker:
+      # Windows tests require "legacy" Docker images
```

Comment:
Do you know why that is?

Reply:
I don't know why this is, but I would guess some kind of difference in the mingw compiler, causing one of the tests to fail.  Good point that I should probably file a new bug to address this later (I don't think it's a high enough priority to do this now).